src/background.js

```diff
@@ -95,5 +94,16 @@ store.subscribe(async ({
         event: 'Lock Wallet'
       })
       break
+
+    case 'ADD_EXTERNAL_CONNECTION':
+      store.dispatch('trackAnalytics', {
+        event: 'Connect to Dapp',
+        properties: {
+          action: `${payload.chain} Dapp connected`,
```

Comment:
i'm not familiar with standard for action,cateogry,label. but I assume these are in the correct format